src/io.ts

```diff
@@ -12,20 +15,49 @@ export const startIOServer = (httpServer: ServerType) => {
     },
   });
 
+  const db = new DbInterface();
+
+  let questions: Array<string> = [];
+  let recvQuestion = 0, playerCount = 0, questionIndex = 0;
+
+  function sendQuestionToPlayers(roomId: string) {
+    db.getQuestion(questions[questionIndex]).then((question) => {
+      io.to(roomId).emit(WS_EVENTS.NEW_QUESTION, question?.Question, Array.from(question?.PossibleAnswers.values()));
+      questionIndex++;
+    });
+  }
+
   // Handle new web socket connection
   io.on("connection", (socket) => {
     console.log(`New client connected: ${socket.id}`);
 
     socket.on(WS_EVENTS.JOIN_ROOM, (room_id: string) => {
       console.log(`Room ID received: ${room_id}`);
+      // join socket.io room
+      socket.join(room_id);
+
+      if (room_id === '1234') {
+        playerCount++;
+        console.log(`Player count: ${playerCount}`);
+        const playerId = `player${playerCount}`;
+        io.to(room_id).emit(WS_EVENTS.NEW_PLAYER, playerId);
+      }
     });
 
     socket.on(WS_EVENTS.NEW_PLAYER, (playerID: string) => {
         console.log(`Player ${playerID} joined the room`);
     });
 
-    socket.on(WS_EVENTS.START_QUIZ, () => {
-        console.log(`${socket.id} started the game!`);
+    socket.on(WS_EVENTS.START_QUIZ, (roomId: string) => {
+      console.log(`${socket.id} started the game!`)
+      db.getQuiz(DEFAULT_QUIZ).then((quiz) => {
+        if (!quiz) {
+            console.error('No quiz found');
+            return;
+        }
+        questions = quiz.Questions;
```

Comment:
[NIT] Since this is just a temp feature implementation, we can use the JS runtime as our way of keeping track of the state of a room. Ideally, we should be using the DB to avoid complexity.